### DISCUSSION TITLE
✅ Show Popup X times per Session

### DISCUSSION THREAD

**Discussion by zeusgr8** [elementor/elementor#20115] [Feature Request]
1. It would be useful if we can set triggers and rules to make the popup appear x number of times per session.
2. we can select the popup, which we want to hide after form submission. (it will be helpful if our subscription form is not on the popup).

**Comment by nicholaszein** [comment]
Hello @zeusgr8!

This is already a feature of the Popup Advanced Rules. Make sure you have the latest version of Elementor Pro installed.

![image](https://github.com/elementor/elementor/assets/113993460/8a6d302d-dffc-4659-afab-93c0b676c5b3)

Kind regards
